webapp/graphite/metrics_filter.py

```diff
@@ -0,0 +1,37 @@
+import re
+
+from django.conf import settings
+from graphite.metric_filter.models import UserMetricFilter
+from graphite.metric_filter.models import GroupMetricFilter
+
+
+def nodeFilter(user, nodePath):
+  if settings.ALLOW_METRICFILTER_BY_USER:
+    if user.id is not None:
+      filters = UserMetricFilter.getFilters(user.id)
+      if filters is not None:
+        for filter in filters.all():
```

Comment:
Minor nit, but please don't ever use variables that override builtin python functions (such as filter). It often leads to much pain and gnashing of teeth when debugging down the road.